app/controllers/genres_controller.rb

```diff
@@ -0,0 +1,26 @@
+
+class GenresController < ApplicationController
+  def index
+    @genres = Genre.all
+    @genre = Genre.new
+  end
+
+  def show
+    @genre = Genre.find(params[:id])
+  end
+
+  def create
+    genre = Genre.new(genre_params)
```

Comment:
by NOT making this an instance variable, I think your render edge case would have broken because the form needs `@genre` to build the fields, etc..